docs/api.yml

```diff
@@ -57,7 +57,7 @@ paths:
             items:
               $ref: "#/definitions/Cluster"
 
-  /clusters?limit={limit}&offset={offset}:
+  /clusters?limit={limit}&offset={offset}&sortby={field}&order=asc:
```

Comment:
and what about provisioners list? Please add it also, and provide the default way if order is not provided

Reply:
and please add notice, that sorting by multiple keys is not supported